onnx/backend/test/case/node/reduce_log_sum.py

```diff
@@ -11,54 +11,67 @@
 class ReduceLogSum(Base):
     @staticmethod
     def export_nokeepdims() -> None:
+        shape = [3, 4, 5]
+        axes = np.array([2, 1], dtype=np.int64)
+
         node = onnx.helper.make_node(
             "ReduceLogSum",
-            inputs=["data"],
+            inputs=["data", "axes"],
             outputs=["reduced"],
-            axes=[2, 1],
             keepdims=0,
         )
-        data = np.random.ranf([3, 4, 5]).astype(np.float32)
-        reduced = np.log(np.sum(data, axis=(2, 1), keepdims=False))
+        data = np.random.ranf(shape).astype(np.float32)
+        reduced = np.log(np.sum(data, axis=tuple(axes), keepdims=False))
         expect(
-            node, inputs=[data], outputs=[reduced], name="test_reduce_log_sum_desc_axes"
+            node,
+            inputs=[data, axes],
+            outputs=[reduced],
+            name="test_reduce_log_sum_desc_axes",
         )
 
+        axes = np.array([0, 1], dtype=np.int64)
         node = onnx.helper.make_node(
             "ReduceLogSum",
-            inputs=["data"],
+            inputs=["data", "axes"],
             outputs=["reduced"],
-            axes=[0, 1],
             keepdims=0,
         )
-        data = np.random.ranf([3, 4, 5]).astype(np.float32)
-        reduced = np.log(np.sum(data, axis=(0, 1), keepdims=False))
+        data = np.random.ranf(shape).astype(np.float32)
+        reduced = np.log(np.sum(data, axis=tuple(axes), keepdims=False))
         expect(
-            node, inputs=[data], outputs=[reduced], name="test_reduce_log_sum_asc_axes"
+            node,
+            inputs=[data, axes],
+            outputs=[reduced],
+            name="test_reduce_log_sum_asc_axes",
         )
 
     @staticmethod
     def export_keepdims() -> None:
         node = onnx.helper.make_node(
-            "ReduceLogSum", inputs=["data"], outputs=["reduced"]
+            "ReduceLogSum", inputs=["data", "axes"], outputs=["reduced"]
         )
         data = np.random.ranf([3, 4, 5]).astype(np.float32)
         reduced = np.log(np.sum(data, keepdims=True))
+        axes = np.array([], dtype=np.int64)
```

Comment:
All Reduce operator now declare axes as optional. I don't think there is any test checking the one input is supported. Should we add one?

Reply:
I am planning to add this type of tests in a later PR. The reason is that ORT has a bug not allowing reduce ops with only one input. I tried with reducesum with ORT and it failed. Because this PR is already too big, I try not to grow it further and to introduce dependencies on ORT.